src/t_hash.c

```diff
@@ -1820,22 +1823,26 @@ static ExpireAction hashTypeActiveExpire(eItem _hashObj, void *ctx) {
     activeExpireCtx->fieldsToExpireQuota -= info.itemsExpired;
 
     /* If hash has no more fields to expire, remove it from HFE DB */
+    ExpireAction ret;
+    robj *key = createStringObject(keystr, sdslen(keystr));
+    notifyKeyspaceEvent(NOTIFY_HASH,"hexpired",key,activeExpireCtx->db->id);
```

Comment:
One note, active expiry may not expire any fields. e.g. we hdel the field with the minimum TTL in the key. 
@moticless correct me if I'm wrong. 

Maybe, we can check `info.itemsExpired` before sending the notification (or calling signalMofiedKey) to be accurate.

Reply:
@tezc @moticless please have a look the commit: [`e9e536a` (#13329)](https://github.com/redis/redis/pull/13329/commits/e9e536a1ab50375f356415feb86bd27b74b67bdc)